packages/dds/tree/api-report/tree.api.md

```diff
@@ -921,63 +909,22 @@ export type InsertableFlexNode<TSchema extends FlexTreeNodeSchema> = FlattenKeys
     readonly [P in string]: InsertableFlexField<TField>;
 } : EmptyObject, TSchema extends LeafNodeSchema<string, infer TValueSchema> ? TValueSchema : undefined, TSchema["name"]>>;
 
-// @beta
-type InsertableObjectFromSchemaRecord<T extends RestrictiveReadonlyRecord<string, ImplicitFieldSchema>> = {
+// @public
+export type InsertableObjectFromSchemaRecord<T extends RestrictiveReadonlyRecord<string, ImplicitFieldSchema>> = {
     readonly [Property in keyof T]: InsertableTreeFieldFromImplicitField<T[Property]>;
 };
 
-// @beta
-type InsertableTreeFieldFromImplicitField<TSchema extends ImplicitFieldSchema = FieldSchema> = TSchema extends FieldSchema<infer Kind, infer Types> ? ApplyKind<InsertableTreeNodeFromImplicitAllowedTypes<Types>, Kind> : TSchema extends ImplicitAllowedTypes_2 ? InsertableTreeNodeFromImplicitAllowedTypes<TSchema> : unknown;
+// @public
+export type InsertableTreeFieldFromImplicitField<TSchema extends ImplicitFieldSchema = FieldSchema> = TSchema extends FieldSchema<infer Kind, infer Types> ? ApplyKind<InsertableTreeNodeFromImplicitAllowedTypes<Types>, Kind> : TSchema extends ImplicitAllowedTypes ? InsertableTreeNodeFromImplicitAllowedTypes<TSchema> : unknown;
 
-// @beta
-export type InsertableTreeNodeFromImplicitAllowedTypes<TSchema extends ImplicitAllowedTypes_2 = TreeNodeSchema> = TSchema extends TreeNodeSchema ? InsertableTypedNode<TSchema> : TSchema extends AllowedTypes_2 ? InsertableTypedNode<FlexListToUnion<TSchema>> : never;
+// @public
+export type InsertableTreeNodeFromImplicitAllowedTypes<TSchema extends ImplicitAllowedTypes = TreeNodeSchema> = TSchema extends TreeNodeSchema ? InsertableTypedNode<TSchema> : TSchema extends AllowedTypes ? InsertableTypedNode<FlexListToUnion<TSchema>> : never;
 
-// @beta
-type InsertableTypedNode<T extends TreeNodeSchema> = (T extends {
+// @public
+export type InsertableTypedNode<T extends TreeNodeSchema> = (T extends {
     implicitlyConstructable: true;
 } ? NodeBuilderData<T> : never) | Unhydrated<NodeFromSchema<T>>;
 
-declare namespace InternalClassTreeTypes {
-    export {
-        ObjectFromSchemaRecord,
-        InsertableObjectFromSchemaRecord,
-        TreeNodeFromImplicitAllowedTypes,
-        ImplicitAllowedTypes_2 as ImplicitAllowedTypes,
-        FieldKind_2 as FieldKind,
-        InsertableTreeFieldFromImplicitField,
-        AllowedTypes_2 as AllowedTypes,
-        FieldSchema,
-        ApplyKind,
-        InsertableTypedNode,
-        NodeBuilderData
-    }
-}
```

Comment:
These namespaces were an attempt to make "transitively public" types less discoverable by re-exporting them as a namespace.  There might be a way to make this work, but it currently runs into issues with API-Extractor when consuming a trimmed rollup.  I've punted and exported the types in the usual way to unblock beta 1.